src/main/java/io/pravega/perf/PravegaStreamHandler.java

```diff
@@ -66,7 +66,6 @@
         this.timeout = timeout;
         this.bgexecutor = bgexecutor;
         streamManager = StreamManager.create(new URI(uri));
-        streamManager.createScope(scope);
```

Comment:
add an option to create the scope; by default , you can disable it.. this code useful for baremetal deployments... not only limited for nautilus deployment.

Reply:
I agree, I would prefer the tool to create the scope if it doesn't exist (as it does currently) BUT provide a way to disable that with an option.